apps/nextjs-app/.eslintrc.js

```diff
@@ -14,6 +14,11 @@ module.exports = {
     project: 'tsconfig.json',
   },
   ignorePatterns: [...getDefaultIgnorePatterns(), '.next', '.out'],
+  settings: {
+    next: {
+      rootDir: 'apps/nextjs-app',
+    },
```

Comment:
Out of curiosity would it work with

```
  settings: {
    next: {
      rootDir: __dirname,
    },
  },
```

? 

That would make it easy to copy/paste